Source/WebKitLegacy/mac/DOM/DOMWheelEvent.mm

```diff
@@ -95,8 +95,7 @@ - (BOOL)isHorizontal
 
 - (void)initWheelEvent:(int)inWheelDeltaX wheelDeltaY:(int)inWheelDeltaY view:(DOMAbstractView *)view screenX:(int)screenX screenY:(int)screenY clientX:(int)clientX clientY:(int)clientY ctrlKey:(BOOL)ctrlKey altKey:(BOOL)altKey shiftKey:(BOOL)shiftKey metaKey:(BOOL)metaKey
 {
-    WebCore::JSMainThreadNullState state;
-    IMPL->initWebKitWheelEvent(inWheelDeltaX, inWheelDeltaY, toWindowProxy(view), screenX, screenY, clientX, clientY, ctrlKey, altKey, shiftKey, metaKey);
+    return;
```

Comment:
Ditto

Reply:
Done in [f9b6161](https://github.com/WebKit/WebKit/commit/f9b6161bb081aba5b98929d04826314245973a5b).